### PR TITLE
Updated Pre-Work Objectives

### PR DESCRIPTION
**Pull request by gildedgardenia**

### CODE REVIEW

File: unit1/pre-work.rst [hunk 1]
```diff
@@ -8,6 +8,7 @@ Goals
    gaining and improving coding skills and knowledge
 - Understand how programming languages allow us to create software
 - Write and run simple programs
+- Understand how to use course systems
```
Comment (CarlyLanglois) [inline unit1/pre-work.rst]: course platforms? is that what this refers to? I'd put in canvas lms, replit, slack

File: unit1/pre-work.rst [hunk 2]
```diff
@@ -19,6 +20,9 @@ How This Course Works
 - Explain how blended learning works
 - Identify the primary activities of the course, and the purpose of
    each: prep work, studios, exercises, and assignments
+- Explain the purpose of each course system and sign up for the different systems
```
Comment (CarlyLanglois) [inline unit1/pre-work.rst]: here again system is too generic for me to know what this refers to. can you add more detail?

**Review by CarlyLanglois — APPROVED** [review]

File: unit1/pre-work.rst (hunk 1 above)
Comment (gildedgardenia) [reply unit1/pre-work.rst]: Done!

File: unit1/pre-work.rst (hunk 2 above)
Comment (gildedgardenia) [reply unit1/pre-work.rst]: Done!

**Pull request merged**